### PR TITLE
[Delivers #103020810] randomize timezone in tests, fix failing test that assumed ET

### PR DESCRIPTION
**Pull request by pkarman**
story: https://www.pivotaltracker.com/story/show/103020810

Seems to be a regression from earlier timezone work. Adds Zonebie gem to randomize timezone on every rspec run, and fixes test that assumed ET.

### CODE REVIEW

File: spec/helpers/value_helper_spec.rb [hunk 1]
```diff
@@ -21,9 +21,11 @@
     end
 
     it "converts time to relative time when specified" do
-      date = DateTime.new(2015,8,15,4,5,6).in_time_zone("Eastern Time (US & Canada)").strftime("%b %-d, %Y at %l:%M%P")
+      date = DateTime.new(2015,8,15,4,5,6).in_time_zone
+      date_str = date.strftime("%b %-d, %Y at %l:%M%P")
+      puts "Time.zone==#{Time.zone}"
```
Comment (cmc333333) [inline spec/helpers/value_helper_spec.rb]: Mind removing the `puts`?

File: spec/helpers/value_helper_spec.rb [hunk 2]
```diff
@@ -21,9 +21,11 @@
...
       relative_date = time_ago_in_words(date)
-      expect(helper.date_with_tooltip(date,true)).to eq('<span title="Aug 15, 2015 at 12:05am">' + relative_date + ' ago</span>')
+      expect(helper.date_with_tooltip(date,true)).to eq('<span title="' + date_str + '">' + relative_date + ' ago</span>')
```
Comment (cmc333333) [inline spec/helpers/value_helper_spec.rb]: I'm not sure this is testing the right thing -- as `date_with_tooltip` calls `strftime`, calling it here doesn't help us much, hey?

**Comment by pkarman** [comment]
thanks @cmc333333, puts zapped.

**Comment by cmc333333** [comment]
Nothing serious holding this back -- if the `to_s` is critical, lmk and I'll merge

**Comment by pkarman** [comment]
I think the `to_s` only affects the tests, since that is where DateTime is being used instead of Time. Everything I've read suggests those 2 classes are interoperable but the code was explicitly checking for `Time`.

**Comment by pkarman** [comment]
the failing tests from circleci are troubling. taking this back for more investigation.

**Comment by pkarman** [comment]
This should be ready to merge now that I got all tests passing again. They were failing anytime the random timezone was +UTC since that resulted in records being created in the "previous year".

File: Gemfile [hunk 3]
```diff
@@ -71,6 +71,7 @@ group :test do
   gem 'simplecov'
   gem 'timecop'
   gem 'turnip'
+  gem 'zonebie'
```
Comment (afeld) [inline Gemfile]: Awesome gem, thanks @alindeman! :us:

**Pull request merged**